lib/matplotlib/texmanager.py

```diff
@@ -68,6 +68,13 @@ class TexManager:
         'computer modern roman': ('cmr', r'\usepackage{type1ec}'),
         'computer modern sans serif': ('cmss', r'\usepackage{type1ec}'),
         'computer modern typewriter': ('cmtt', r'\usepackage{type1ec}')}
+    _font_types = {
```

Comment:
This all looks good, but its a pretty big change with very minimal documentation.  Sorry if I missed the discussion, but was that on-purpose or should this be documented more thoroughly?

Reply:
I added some lines on the `tutorials/text/usetex.py` but maybe isn't enough? Should I write some more detailed docs or add an example in the `/gallery/text_labels_and_annotations/tex_demo.py`?

Reply:
Are font families not explained elsewhere?  If they are we can add there?   If not, then maybe we need something?

Reply:
To the best of my knowledge, font families are explained in `examples/text_labels_and_annotations/font_family_rc_sgskip.py` but, in Latex context, I think that the only docs is the `tutorials/text/usetex.py` and `examples/text_labels_and_annotations/tex_demo.py`.  Maybe @anntzer knows some other file.

Reply:
Would you be willing to open an issue if there really is nowhere else this is explained?  I don't think it's reasonable to ask you to fix this as part of this PR but it seems we are asked about font selection a _lot_ for it not to be explained clearly somewhere obvious.

Reply:
Absolutely. I definitely could try to do it.

Reply:
https://matplotlib.org/stable/tutorials/text/text_props.html#default-font

Reply:
I agree with @aitikgupta, maybe it's a better idea to make this new notation available to be used with all the others text renders before changing the `tutorials/text/text_props.py`  docs?

Reply:
Maybe it's a better idea make this new notation available to be used with the other text renders before changing the `tutorials/text/text_props.py`? That being said, maybe only  `/tutorials/text/usetex.py` is a feasible documenting (_for now_), like @aitikgupta proposed.